app/scripts/modules/core/authentication/authentication.service.ts

```diff
@@ -3,13 +3,15 @@ import {module} from 'angular';
 export interface IUser {
   name: string;
   authenticated: boolean;
+  roles: string[];
```

Comment:
should be optional, `roles?: string[]` - also, `string` is fine for now, but I could see this being a more complex type at some point.